### ISSUE TITLE
New Rule: no-missing-dependent-keys

### ISSUE THREAD

**Issue by mhluska** [ember-cli/eslint-plugin-ember#307]
It would be useful if there was a way to lint the following:

bad:

```js
prop: computed('dogs', function() {
  return this.get('dogs') + this.get('cats');
}),
```

good:

```js
prop: computed('dogs', 'cats', function() {
  return this.get('dogs') + this.get('cats');
}),
```

I often run into this situation where a dependent key is not explicitly listed and the prop doesn't update correctly.

**Comment by rwjblue** [comment]
Seems reasonable to me...

**Comment by bmish** [comment]
I submitted an implementation in #458, but it doesn't handle ES5 getter usage, and I'm not sure it's practical to handle that unfortunately, so the rule may not be worthwhile anymore?

**Comment by bmish** [comment]
I submitted a full implementation of this in #458.
